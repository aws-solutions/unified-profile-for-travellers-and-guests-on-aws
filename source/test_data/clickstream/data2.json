{"modelVersion":"1.0","event_type":"SelectOrigin","event_timestamp":"2023-04-04T21:37:19.970150076Z","arrival_timestamp":0,"event_version":"","application":{"id":""},"client":{"id":""},"device":{"useragent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/107.0.0.0 Safari/537.36"},"session":{"id":"852a397e-ad1e-42e5-9ccd-35fd4a8c7cb1"},"attributes":[{"type":"string","name":"quantities","stringValue":"","stringValues":null,"numValue":0.0,"numValues":[2.0,4.0,4.0,1.0,3.0,2.0]},{"type":"strings","name":"products","stringValue":"","stringValues":["KING_SUPER_SAVER","DBL-BAR","CHMPGN","KING_SUPER_SAVER","KING_SUPER_SAVER","DBL-BAR"],"numValue":0.0,"numValues":null},{"type":"string","name":"destination","stringValue":"Mesa, Utah, Wallis and Futuna","stringValues":null,"numValue":0.0,"numValues":null},{"type":"number","name":"num_nights","stringValue":"","stringValues":null,"numValue":6.0,"numValues":null},{"type":"string","name":"room_type","stringValue":"KING","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"rate_plan","stringValue":"NON_REFUNDABLE","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"checkin_date","stringValue":"2023-08-08","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"checkout_date","stringValue":"2023-08-14","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"hotel_code","stringValue":"PAR17","stringValues":null,"numValue":0.0,"numValues":null},{"type":"strings","name":"hotel_code_list","stringValue":"","stringValues":["LON19","PAR17","NYC01","ROM09","ATL05","ATL01"],"numValue":0.0,"numValues":null}],"endpoint":{"id":""},"awsAccountId":"","userId":""}{"modelVersion":"1.0","event_type":"SelectExperience","event_timestamp":"2023-04-04T21:37:24.970150076Z","arrival_timestamp":0,"event_version":"","application":{"id":""},"client":{"id":""},"device":{"useragent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/107.0.0.0 Safari/537.36"},"session":{"id":"852a397e-ad1e-42e5-9ccd-35fd4a8c7cb1"},"attributes":[{"type":"string","name":"quantities","stringValue":"","stringValues":null,"numValue":0.0,"numValues":[3.0,2.0,2.0]},{"type":"strings","name":"products","stringValue":"","stringValues":["CHMPGN","SUITE-BAR","BKFST"],"numValue":0.0,"numValues":null},{"type":"string","name":"destination","stringValue":"Mesa, Utah, Wallis and Futuna","stringValues":null,"numValue":0.0,"numValues":null},{"type":"number","name":"num_nights","stringValue":"","stringValues":null,"numValue":6.0,"numValues":null},{"type":"string","name":"room_type","stringValue":"SUITE","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"rate_plan","stringValue":"NON_REFUNDABLE","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"checkin_date","stringValue":"2023-08-08","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"checkout_date","stringValue":"2023-08-14","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"hotel_code","stringValue":"ATL06","stringValues":null,"numValue":0.0,"numValues":null},{"type":"strings","name":"hotel_code_list","stringValue":"","stringValues":["NYC05","NYC01","NYC16"],"numValue":0.0,"numValues":null}],"endpoint":{"id":""},"awsAccountId":"","userId":""}{"modelVersion":"1.0","event_type":"SelectEndTime","event_timestamp":"2023-04-04T21:37:35.970150076Z","arrival_timestamp":0,"event_version":"","application":{"id":""},"client":{"id":""},"device":{"useragent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/107.0.0.0 Safari/537.36"},"session":{"id":"852a397e-ad1e-42e5-9ccd-35fd4a8c7cb1"},"attributes":[{"type":"string","name":"quantities","stringValue":"","stringValues":null,"numValue":0.0,"numValues":[2.0,5.0,4.0,4.0,1.0,1.0]},{"type":"strings","name":"products","stringValue":"","stringValues":["DBL-BAR","CHMPGN","BKFST","SUITE-BAR","DBL-BAR","CHMPGN"],"numValue":0.0,"numValues":null},{"type":"string","name":"destination","stringValue":"Mesa, Utah, Wallis and Futuna","stringValues":null,"numValue":0.0,"numValues":null},{"type":"number","name":"num_nights","stringValue":"","stringValues":null,"numValue":6.0,"numValues":null},{"type":"string","name":"room_type","stringValue":"KING","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"rate_plan","stringValue":"NON_REFUNDABLE","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"checkin_date","stringValue":"2023-08-08","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"checkout_date","stringValue":"2023-08-14","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"hotel_code","stringValue":"TOK19","stringValues":null,"numValue":0.0,"numValues":null},{"type":"strings","name":"hotel_code_list","stringValue":"","stringValues":["ROM09","ROM06","LON18","SPO15","ROM11","LIM06"],"numValue":0.0,"numValues":null}],"endpoint":{"id":""},"awsAccountId":"","userId":""}{"modelVersion":"1.0","event_type":"SearchAddOn","event_timestamp":"2023-04-04T21:37:41.970150076Z","arrival_timestamp":0,"event_version":"","application":{"id":""},"client":{"id":""},"device":{"useragent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/107.0.0.0 Safari/537.36"},"session":{"id":"852a397e-ad1e-42e5-9ccd-35fd4a8c7cb1"},"attributes":[{"type":"string","name":"quantities","stringValue":"","stringValues":null,"numValue":0.0,"numValues":[2.0,4.0,2.0,1.0]},{"type":"strings","name":"products","stringValue":"","stringValues":["KING_SUPER_SAVER","SUITE-BAR","CHMPGN","SUITE-BAR"],"numValue":0.0,"numValues":null},{"type":"string","name":"destination","stringValue":"Mesa, Utah, Wallis and Futuna","stringValues":null,"numValue":0.0,"numValues":null},{"type":"number","name":"num_nights","stringValue":"","stringValues":null,"numValue":6.0,"numValues":null},{"type":"string","name":"room_type","stringValue":"KING","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"rate_plan","stringValue":"NON_REFUNDABLE","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"checkin_date","stringValue":"2023-08-08","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"checkout_date","stringValue":"2023-08-14","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"hotel_code","stringValue":"ATL06","stringValues":null,"numValue":0.0,"numValues":null},{"type":"strings","name":"hotel_code_list","stringValue":"","stringValues":["ROM01","LON15","PAR08","LON04"],"numValue":0.0,"numValues":null}],"endpoint":{"id":""},"awsAccountId":"","userId":""}{"modelVersion":"1.0","event_type":"ConfirmBooking","event_timestamp":"2023-04-04T21:37:55.970150076Z","arrival_timestamp":0,"event_version":"","application":{"id":""},"client":{"id":""},"device":{"useragent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/107.0.0.0 Safari/537.36"},"session":{"id":"852a397e-ad1e-42e5-9ccd-35fd4a8c7cb1"},"attributes":[{"type":"string","name":"quantities","stringValue":"","stringValues":null,"numValue":0.0,"numValues":[4.0,5.0,2.0,3.0,3.0,2.0,2.0,4.0,3.0]},{"type":"strings","name":"products","stringValue":"","stringValues":["KING_SUPER_SAVER","SUITE-BAR","SUITE-BAR","CHMPGN","SUITE-BAR","KING_SUPER_SAVER","SUITE-BAR","KING_SUPER_SAVER","KING_SUPER_SAVER"],"numValue":0.0,"numValues":null},{"type":"string","name":"destination","stringValue":"Mesa, Utah, Wallis and Futuna","stringValues":null,"numValue":0.0,"numValues":null},{"type":"number","name":"num_nights","stringValue":"","stringValues":null,"numValue":6.0,"numValues":null},{"type":"string","name":"room_type","stringValue":"KING","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"rate_plan","stringValue":"NON_REFUNDABLE","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"checkin_date","stringValue":"2023-08-08","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"checkout_date","stringValue":"2023-08-14","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"hotel_code","stringValue":"LIM19","stringValues":null,"numValue":0.0,"numValues":null},{"type":"strings","name":"hotel_code_list","stringValue":"","stringValues":["PAR06","PAR11","LON09","ATL11","ATL08","ROM13","SGP02","TOK09","LON19"],"numValue":0.0,"numValues":null}],"endpoint":{"id":""},"awsAccountId":"","userId":""}{"modelVersion":"1.0","event_type":"SelectAddOn","event_timestamp":"2023-04-04T21:38:03.970150076Z","arrival_timestamp":0,"event_version":"","application":{"id":""},"client":{"id":""},"device":{"useragent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/107.0.0.0 Safari/537.36"},"session":{"id":"852a397e-ad1e-42e5-9ccd-35fd4a8c7cb1"},"attributes":[{"type":"string","name":"quantities","stringValue":"","stringValues":null,"numValue":0.0,"numValues":[4.0,2.0,5.0,4.0,3.0,2.0]},{"type":"strings","name":"products","stringValue":"","stringValues":["KING_SUPER_SAVER","DBL-BAR","SUITE-BAR","SUITE-BAR","DBL-BAR","KING_SUPER_SAVER"],"numValue":0.0,"numValues":null},{"type":"string","name":"destination","stringValue":"Mesa, Utah, Wallis and Futuna","stringValues":null,"numValue":0.0,"numValues":null},{"type":"number","name":"num_nights","stringValue":"","stringValues":null,"numValue":6.0,"numValues":null},{"type":"string","name":"room_type","stringValue":"DBL","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"rate_plan","stringValue":"NON_REFUNDABLE","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"checkin_date","stringValue":"2023-08-08","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"checkout_date","stringValue":"2023-08-14","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"hotel_code","stringValue":"PAR15","stringValues":null,"numValue":0.0,"numValues":null},{"type":"strings","name":"hotel_code_list","stringValue":"","stringValues":["SPO15","LON15","ROM04","TOK18","LIM00","LIM17"],"numValue":0.0,"numValues":null}],"endpoint":{"id":""},"awsAccountId":"","userId":""}{"modelVersion":"1.0","event_type":"SearchDestination","event_timestamp":"2023-04-04T21:38:12.970150076Z","arrival_timestamp":0,"event_version":"","application":{"id":""},"client":{"id":""},"device":{"useragent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/107.0.0.0 Safari/537.36"},"session":{"id":"852a397e-ad1e-42e5-9ccd-35fd4a8c7cb1"},"attributes":[{"type":"string","name":"quantities","stringValue":"","stringValues":null,"numValue":0.0,"numValues":[4.0,5.0,2.0,5.0,4.0,5.0,4.0,1.0]},{"type":"strings","name":"products","stringValue":"","stringValues":["DBL-BAR","BKFST","KING_SUPER_SAVER","BKFST","CHMPGN","KING_SUPER_SAVER","BKFST","BKFST"],"numValue":0.0,"numValues":null},{"type":"string","name":"destination","stringValue":"Mesa, Utah, Wallis and Futuna","stringValues":null,"numValue":0.0,"numValues":null},{"type":"number","name":"num_nights","stringValue":"","stringValues":null,"numValue":6.0,"numValues":null},{"type":"string","name":"room_type","stringValue":"SUITE","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"rate_plan","stringValue":"NON_REFUNDABLE","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"checkin_date","stringValue":"2023-08-08","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"checkout_date","stringValue":"2023-08-14","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"hotel_code","stringValue":"PAR15","stringValues":null,"numValue":0.0,"numValues":null},{"type":"strings","name":"hotel_code_list","stringValue":"","stringValues":["LON09","PAR07","SPO15","ROM16","ATL00","TOK19","SGP03","SPO19"],"numValue":0.0,"numValues":null}],"endpoint":{"id":""},"awsAccountId":"","userId":""}{"modelVersion":"1.0","event_type":"SearchFlight","event_timestamp":"2023-04-04T21:38:27.970150076Z","arrival_timestamp":0,"event_version":"","application":{"id":""},"client":{"id":""},"device":{"useragent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/107.0.0.0 Safari/537.36"},"session":{"id":"852a397e-ad1e-42e5-9ccd-35fd4a8c7cb1"},"attributes":[{"type":"string","name":"quantities","stringValue":"","stringValues":null,"numValue":0.0,"numValues":[5.0,4.0,4.0,3.0,2.0,1.0]},{"type":"strings","name":"products","stringValue":"","stringValues":["KING_SUPER_SAVER","SUITE-BAR","CHMPGN","CHMPGN","KING_SUPER_SAVER","SUITE-BAR"],"numValue":0.0,"numValues":null},{"type":"string","name":"destination","stringValue":"Mesa, Utah, Wallis and Futuna","stringValues":null,"numValue":0.0,"numValues":null},{"type":"number","name":"num_nights","stringValue":"","stringValues":null,"numValue":6.0,"numValues":null},{"type":"string","name":"room_type","stringValue":"KING","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"rate_plan","stringValue":"NON_REFUNDABLE","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"checkin_date","stringValue":"2023-08-08","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"checkout_date","stringValue":"2023-08-14","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"hotel_code","stringValue":"SGP07","stringValues":null,"numValue":0.0,"numValues":null},{"type":"strings","name":"hotel_code_list","stringValue":"","stringValues":["SPO17","SGP04","LON03","SGP14","LON10","SPO14"],"numValue":0.0,"numValues":null}],"endpoint":{"id":""},"awsAccountId":"","userId":""}{"modelVersion":"1.0","event_type":"RetreiveBooking","event_timestamp":"2023-04-04T21:38:38.970150076Z","arrival_timestamp":0,"event_version":"","application":{"id":""},"client":{"id":""},"device":{"useragent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/107.0.0.0 Safari/537.36"},"session":{"id":"852a397e-ad1e-42e5-9ccd-35fd4a8c7cb1"},"attributes":[{"type":"string","name":"quantities","stringValue":"","stringValues":null,"numValue":0.0,"numValues":[4.0,2.0]},{"type":"strings","name":"products","stringValue":"","stringValues":["CHMPGN","BKFST"],"numValue":0.0,"numValues":null},{"type":"string","name":"destination","stringValue":"Mesa, Utah, Wallis and Futuna","stringValues":null,"numValue":0.0,"numValues":null},{"type":"number","name":"num_nights","stringValue":"","stringValues":null,"numValue":6.0,"numValues":null},{"type":"string","name":"room_type","stringValue":"SUITE","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"rate_plan","stringValue":"SUPER_SAVER","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"checkin_date","stringValue":"2023-08-08","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"checkout_date","stringValue":"2023-08-14","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"hotel_code","stringValue":"SPO15","stringValues":null,"numValue":0.0,"numValues":null},{"type":"strings","name":"hotel_code_list","stringValue":"","stringValues":["ATL06","SPO16"],"numValue":0.0,"numValues":null}],"endpoint":{"id":""},"awsAccountId":"","userId":""}{"modelVersion":"1.0","event_type":"StartSession","event_timestamp":"2023-04-04T21:38:46.970150076Z","arrival_timestamp":0,"event_version":"","application":{"id":""},"client":{"id":""},"device":{"useragent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/107.0.0.0 Safari/537.36"},"session":{"id":"852a397e-ad1e-42e5-9ccd-35fd4a8c7cb1"},"attributes":[{"type":"string","name":"quantities","stringValue":"","stringValues":null,"numValue":0.0,"numValues":[5.0,5.0,5.0,3.0,5.0,4.0,2.0]},{"type":"strings","name":"products","stringValue":"","stringValues":["CHMPGN","SUITE-BAR","SUITE-BAR","CHMPGN","SUITE-BAR","SUITE-BAR","SUITE-BAR"],"numValue":0.0,"numValues":null},{"type":"string","name":"destination","stringValue":"Mesa, Utah, Wallis and Futuna","stringValues":null,"numValue":0.0,"numValues":null},{"type":"number","name":"num_nights","stringValue":"","stringValues":null,"numValue":6.0,"numValues":null},{"type":"string","name":"room_type","stringValue":"DBL","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"rate_plan","stringValue":"NON_REFUNDABLE","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"checkin_date","stringValue":"2023-08-08","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"checkout_date","stringValue":"2023-08-14","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"hotel_code","stringValue":"LIM16","stringValues":null,"numValue":0.0,"numValues":null},{"type":"strings","name":"hotel_code_list","stringValue":"","stringValues":["SPO16","NYC06","ATL07","ATL09","TOK06","SPO03","LON07"],"numValue":0.0,"numValues":null}],"endpoint":{"id":""},"awsAccountId":"","userId":""}{"modelVersion":"1.0","event_type":"SearchSingleAvailability","event_timestamp":"2023-04-04T21:38:54.970150076Z","arrival_timestamp":0,"event_version":"","application":{"id":""},"client":{"id":""},"device":{"useragent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/107.0.0.0 Safari/537.36"},"session":{"id":"852a397e-ad1e-42e5-9ccd-35fd4a8c7cb1"},"attributes":[{"type":"string","name":"quantities","stringValue":"","stringValues":null,"numValue":0.0,"numValues":[1.0]},{"type":"strings","name":"products","stringValue":"","stringValues":["KING_SUPER_SAVER"],"numValue":0.0,"numValues":null},{"type":"string","name":"destination","stringValue":"Mesa, Utah, Wallis and Futuna","stringValues":null,"numValue":0.0,"numValues":null},{"type":"number","name":"num_nights","stringValue":"","stringValues":null,"numValue":6.0,"numValues":null},{"type":"string","name":"room_type","stringValue":"KING","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"rate_plan","stringValue":"SUPER_SAVER","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"checkin_date","stringValue":"2023-08-08","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"checkout_date","stringValue":"2023-08-14","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"hotel_code","stringValue":"LIM02","stringValues":null,"numValue":0.0,"numValues":null},{"type":"strings","name":"hotel_code_list","stringValue":"","stringValues":["SGP05"],"numValue":0.0,"numValues":null}],"endpoint":{"id":""},"awsAccountId":"","userId":""}{"modelVersion":"1.0","event_type":"SearchAddOn","event_timestamp":"2023-04-04T21:39:05.970150076Z","arrival_timestamp":0,"event_version":"","application":{"id":""},"client":{"id":""},"device":{"useragent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/107.0.0.0 Safari/537.36"},"session":{"id":"852a397e-ad1e-42e5-9ccd-35fd4a8c7cb1"},"attributes":[{"type":"string","name":"quantities","stringValue":"","stringValues":null,"numValue":0.0,"numValues":[3.0,5.0,4.0,4.0,4.0,1.0,4.0,3.0,1.0]},{"type":"strings","name":"products","stringValue":"","stringValues":["KING_SUPER_SAVER","KING_SUPER_SAVER","KING_SUPER_SAVER","BKFST","BKFST","SUITE-BAR","DBL-BAR","CHMPGN","BKFST"],"numValue":0.0,"numValues":null},{"type":"string","name":"destination","stringValue":"Mesa, Utah, Wallis and Futuna","stringValues":null,"numValue":0.0,"numValues":null},{"type":"number","name":"num_nights","stringValue":"","stringValues":null,"numValue":6.0,"numValues":null},{"type":"string","name":"room_type","stringValue":"KING","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"rate_plan","stringValue":"BAR","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"checkin_date","stringValue":"2023-08-08","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"checkout_date","stringValue":"2023-08-14","stringValues":null,"numValue":0.0,"numValues":null},{"type":"string","name":"hotel_code","stringValue":"ROM10","stringValues":null,"numValue":0.0,"numValues":null},{"type":"strings","name":"hotel_code_list","stringValue":"","stringValues":["ROM08","LON19","TOK07","LON15","ATL02","TOK18","LIM19","LIM07","LON14"],"numValue":0.0,"numValues":null}],"endpoint":{"id":""},"awsAccountId":"","userId":""}