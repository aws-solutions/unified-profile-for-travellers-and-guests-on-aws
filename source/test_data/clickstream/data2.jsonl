{"modelVersion":"1.0","event_type":"cross_geofence","application":{"id":""},"client":{"id":""},"device":{"useragent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/107.0.0.0 Safari/537.36","ipAddress":"80.45.131.233"},"session":{"id":"39350c2e-db45-4cc6-9e6b-d3bd7b83098d"},"attributes":[{"type":"string","name":"geofence_latitude","stringValue":"-49.830948"},{"type":"string","name":"geofence_longitude","stringValue":"168.371146"},{"type":"string","name":"geofence_name","stringValue":"Restaurant"},{"type":"string","name":"geofence_id","stringValue":"restaurant_1"}],"endpoint":{},"event_timestamp":"2024-02-13T01:07:25.877850Z","arrival_timestamp":"0001-01-01T00:00:00.000000Z"}
{"modelVersion":"1.0","event_type":"start_session","application":{"id":""},"client":{"id":""},"device":{"useragent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/107.0.0.0 Safari/537.36","ipAddress":"144.19.215.212"},"session":{"id":"39350c2e-db45-4cc6-9e6b-d3bd7b83098d"},"attributes":[{"type":"string","name":"customer_phone","stringValue":"42 3142275506"},{"type":"string","name":"loyalty_id","stringValue":"6378611383"},{"type":"string","name":"customer_gender","stringValue":"other"},{"type":"string","name":"customer_id","stringValue":"0368859475"},{"type":"string","name":"language_code","stringValue":"ba"},{"type":"string","name":"customer_birthdate","stringValue":"2017-02-21"},{"type":"string","name":"customer_first_name","stringValue":"Max"},{"type":"string","name":"customer_last_name","stringValue":"Lowe"},{"type":"numbers","name":"quantities","numValues":[2.0,1.0]},{"type":"strings","name":"products","stringValues":["FARE","FARE"]},{"type":"numbers","name":"products_prices","numValues":[921.0,474.0]},{"type":"string","name":"fare_class","stringValue":"J"},{"type":"string","name":"fare_type","stringValue":"P"},{"type":"strings","name":"flight_segments_departure_date_time","stringValues":["2024-11-16T13:01:21"]},{"type":"strings","name":"flight_segments_arrival_date_time","stringValues":["2024-11-17T00:01:21"]},{"type":"strings","name":"flight_segment_sku","stringValues":["EK4893"]},{"type":"string","name":"flight_market","stringValue":"DOM,DOM"},{"type":"string","name":"flight_type","stringValue":"round-trip"},{"type":"string","name":"origin_date","stringValue":"DOM,DOM"},{"type":"string","name":"origin_date_time","stringValue":"DOM,DOM"},{"type":"string","name":"returning_date_time","stringValue":"DOM,DOM"},{"type":"string","name":"returning_date","stringValue":"DOM,DOM"},{"type":"string","name":"returning_flight_route","stringValue":"DOM,DOM"},{"type":"number","name":"num_pax_adults","numValue":2.0},{"type":"number","name":"num_pax_inf","numValue":1.0},{"type":"number","name":"num_pax_children","numValue":1.0},{"type":"string","name":"pax_type","stringValue":"child"},{"type":"number","name":"total_passengers","numValue":4.0}],"endpoint":{},"alternateProfileIds":[{"name":"troop","value":"808df6d8-bc5d-4a3e-8af2-c2218d3ae5d2","description":"Voluptas et veritatis maxime adipisci.","lastUpdatedOn":"1958-05-12T12:56:06.242472Z"},{"name":"library","value":"5bac7251-ff88-4aa2-a4d2-7b47687dc607","description":"Repellat non quasi eum adipisci.","lastUpdatedOn":"1917-06-02T21:00:27.398835Z"}],"event_timestamp":"2024-02-13T01:07:30.877850Z","arrival_timestamp":"0001-01-01T00:00:00.000000Z"}
