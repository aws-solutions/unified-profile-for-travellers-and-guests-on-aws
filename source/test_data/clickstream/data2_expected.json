{"model_version": "1.0", "object_type": "clickstream_event", "event_type": "SelectOrigin", "event_timestamp": "2023-04-04T21:37:19.970150076Z", "arrival_timestamp": 0, "event_version": "", "user_agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/107.0.0.0 Safari/537.36", "session_id": "852a397e-ad1e-42e5-9ccd-35fd4a8c7cb1", "traveller_id": "852a397e-ad1e-42e5-9ccd-35fd4a8c7cb1", "destination": "Mesa, Utah, Wallis and Futuna", "checkin_date": "2023-08-08", "products": "KING_SUPER_SAVER|DBL-BAR|CHMPGN|KING_SUPER_SAVER|KING_SUPER_SAVER|DBL-BAR", "num_nights": 6.0, "room_type": "KING", "rate_plan": "NON_REFUNDABLE", "checkout_date": "2023-08-14", "hotel_code": "PAR17", "hotel_code_list": "LON19|PAR17|NYC01|ROM09|ATL05|ATL01", "aws_account_id": "", "last_updated": "2023-04-04T21:37:19.970150076Z", "last_updated_partition": "2023/04/04/21/37/19/970150076Z"}